apps/web/src/features/subjects/pages/SubjectGraphPage.tsx

```diff
@@ -72,12 +124,23 @@ export const SubjectGraphPage = () => {
               />
             </div>
           </div>
-          <div data-cy="time-select">
-            <TimeDropdown setMinTime={setMinDate} />
+          <div className="flex flex-col gap-2 lg:flex-row">
+            <div data-cy="time-select">
+              <TimeDropdown setMinTime={setMinDate} />
+            </div>
+            <div className="relative w-full whitespace-nowrap" data-cy="download-button">
+              <Button
+                className="relative w-full whitespace-nowrap text-sm"
+                disabled={!instrument}
+                label="Download"
+                variant="secondary"
+                onClick={() => void handleGraphDownload()}
+              />
+            </div>
           </div>
         </div>
       </div>
-      <div>
+      <div className="print:h-[400px] print:w-[400px]" ref={graphRef}>
```

Comment:
Also, please rename `graphRef` to `graphContainerRef` or something, since it does not actually reference the graph and this could lead to confusion.